File: C1.UWP.FlexChart/VB/WealthHealth/Converters.vb
Imports Windows.UI.Xaml.Data
Imports System.Globalization
Imports System

Public Class IntToStringConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return value.ToString()
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Dim val As Object = Double.Parse(value.ToString())
        Return CType(Math.Floor(val), Integer)
    End Function
End Class

File: C1.UWP.FlexChart/CS/WealthHealth/Converters.cs
using System;
using System.Globalization;
using Windows.UI.Xaml.Data;

namespace WealthHealth
{
    public class IntToStringConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            return value.ToString();
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            var val = double.Parse(value.ToString());
            return (int)Math.Floor(val);
        }
    }
}
